cpp/src/parquet/encryption/test_encryption_util.cc

```diff
@@ -334,7 +335,10 @@ void FileDecryptor::DecryptFile(
     reader_properties.file_decryption_properties(file_decryption_properties->DeepClone());
   }
 
-  auto file_reader = parquet::ParquetFileReader::OpenFile(file, false, reader_properties);
+  std::shared_ptr<::arrow::io::RandomAccessFile> source;
+  PARQUET_ASSIGN_OR_THROW(
+      source, ::arrow::io::ReadableFile::Open(file, reader_properties.memory_pool()));
```

Comment:
I was having issues with the encryption tests on Windows, which would flake because the temp Parquet file was still in use when it deleted it. This was an attempt at fixing that.

Reply:
Fair enough, thanks :-)